plugins/w3c_logs.yaml

```diff
@@ -35,6 +35,41 @@ parameters:
     description: Max number of W3C files that will be open during a polling cycle
     type: int
     default: 512
+  - name: timestamp_layout
+    description: Optional timestamp layout which will parse a timestamp field
+    type: string
+  - name: timestamp_parse_from
+    description: Field to parse the timestamp from, required if timestamp_layout is set
+    type: string
+  - name: timestamp_layout_type
+    description: Optional timestamp layout type for parsing the timestamp, suggested if timestamp_layout is set
+    type: string
```

Comment:
What are the possible values for this?

Reply:
Thanks Andy, I'll get the three values enumerated.